generators/utils.js

```diff
@@ -454,39 +454,56 @@ function getJavadoc(text, indentSize) {
 /**
  * Build an enum object
  * @param {any} field : entity field
- * @param {string} angularAppName
- * @param {string} packageName
  * @param {string} clientRootFolder
  */
-function buildEnumInfo(field, angularAppName, packageName, clientRootFolder) {
+function buildEnumInfo(field, clientRootFolder) {
```

Comment:
I think we should keep api stability for utils.

Reply:
This isn't used at all... The generator is full of this

Reply:
I'll create a second one, cleaned up and deprecate the other, so as to be removed for v7.